src/constants/DefaultFields.ts

```diff
@@ -24,6 +24,20 @@ export const DefaultFields: Record<string, DefaultField> = Object.freeze({
     question_type: QuestionTypes.MULTIPLE_CHOICE,
     options: [],
   },
+  CURRENT_SESSION: {
+    id: DefaultFieldKey.CURRENT_SESSION,
+    is_default: true,
+    name: "Session",
+    question_type: QuestionTypes.MULTIPLE_CHOICE,
+    options: [],
+  },
+  CURRENT_PREFERRED_CLASS: {
+    id: DefaultFieldKey.CURRENT_PREFERRED_CLASS,
+    is_default: true,
```

Comment:
this column isn't visible by default in the designs, so we should set this as false